machine_management/creating-infrastructure-machinesets.adoc

```diff
@@ -37,6 +37,8 @@ Use the sample MachineSet for your cloud.
 
 include::modules/machineset-yaml-aws.adoc[leveloffset=+3]
 
+AWS MachineSets support non-guaranteed, xref:../machine_management/creating_machinesets/creating-machineset-aws.html#machineset-non-guaranteed-instance_creating-machineset-aws[Spot Instances], which can save you costs compared to On-Demand Instance prices. You can xref:../machine_management/creating_machinesets/creating-machineset-aws.html#machineset-creating-non-guaranteed-instance_creating-machineset-aws[configure Spot Instances] by adding SpotMarketOptions to the MachineSet YAML file.
```

Comment:
@lbarbeevargas the comma after `non-guaranteed` threw me off. Do you think it makes sense to remove that comma? Maybe I'm just reading it wrong. Your call :slightly_smiling_face: